sym/assets/dashboards/sym_overview.json

```diff
@@ -0,0 +1,419 @@
+{
+    "title": "Sym",
+    "description": null,
```

Comment:
Please add a short description describing what a customer can expect in this dashboard.